projects/m2/026-play-bingo/python/main.py

```diff
@@ -0,0 +1,43 @@
+from random import shuffle
+from create_bingo import*
+from winning_card import*
```

Comment:
forma assolutamente sconsigliata di import poiché non permette di capire di capire le funzioni che utilizzi da quale modulo provengono, inoltre ci sono casi in cui i nomi delle funzioni sono uguali in più moduli/librerie, in quel caso neanche l'interprete Python riesce a distinguerli, pertanto inserisci possibili bug nel codice senza accorgertene